distributed/comm/utils.py

```diff
@@ -40,15 +39,17 @@ def _to_frames():
             logger.exception(e)
             raise
 
-    if FRAME_OFFLOAD_THRESHOLD and allow_offload:
+    # Offload serializing large frames to improve event loop responsiveness.
+    offload_threshold = get_offload_threshold()
+    if offload_threshold and allow_offload:
```

Comment:
Hrm, this may be expensive to do on every message. Thoughts?

Reply:
It looks like this adds ~3ms:

```python
In [1]: import dask

In [2]: from distributed.comm.utils import get_offload_threshold

In [3]: %timeit get_offload_threshold()
2.96 µs ± 44.7 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)
```

This change was so we could set `distributed.comm.offload` at runtime (i.e. `with dask.config.set(distributed__comm__offload=1):`) instead of just at startup time. Though I agree since is run for every message, the increased overhead may not be worth the extra flexibility.

Reply:
I reverted the addition of `get_offload_threshold` and we now just pull in `distributed.comm.offload` at startup time